docs/getting-started/setup.md

```diff
@@ -73,5 +73,6 @@ Alternatively you can build the node by yourself.
 This can take quite a while though!
 
 ```bash
-cargo install contracts-node --git https://github.com/paritytech/substrate-contracts-node.git --tag v0.23.0 --force --locked
-```
\ No newline at end of file
+git clone https://github.com/paritytech/substrate-contracts-node.git
```

Comment:
You don't need to clone the source here, the below `cargo install` fetches the latest published version from crates.io

Reply:
does it build from source?

Reply:
You can build it from source, yes: either by cloning and then `cargo build --release` or by using the `--git` source for `cargo install` as per the previous command.

However we recommend installing the latest version from `crates.io` nowadays.

Reply:
@ascjones therefore, if it does not install from source, then this edit seems to be incorrect.
I was editing section that precisely was describing how to build from source. Should we revert this PR?

Reply:
Well, `cargo install` does just download source from crates.io and builds it locally. Usually the latest `ink` will only target the latest release of `substrate-contracts-node` so this is okay, if not entirely clear.

Reply:
OK, I misunderstood previous comment then. Should be ok then.